ext/tokenizer/tokenizer.c

```diff
@@ -106,6 +106,10 @@ PHP_METHOD(PhpToken, tokenize)
 		zend_throw_error(NULL, "Cannot instantiate abstract class %s", ZSTR_VAL(token_class->name));
 		RETURN_THROWS();
 	}
+	if (token_class->ce_flags & ZEND_ACC_STATIC) {
```

Comment:
This condition is impossible, the token class cannot be static.

Reply:
I don't know why this change was made by @oplanre. Do you suppose we can just remove it?

Reply:
You can remove it, or replace it with an assert. Since with this RFC static classes cannot extend non-static ones, and `PhpToken` is non-static, this condition is never true.